crates/consensus/Cargo.toml

```
@@ -11,14 +11,19 @@ readme = "README.md"
 reth-primitives = { path = "../primitives" }
 reth-interfaces = { path = "../interfaces" }
 reth-provider = { path = "../storage/provider" }
-reth-rlp = {path = "../common/rlp"}
+reth-rlp = { path = "../common/rlp" }
+reth-rpc-types = { path = "../net/rpc-types" }
 
-# common
+# async
+futures = "0.3"
 async-trait = "0.1.57"
+tokio = { version = "1.21.2", features = ["sync"] }
```

consistent tokio versioning, noticed it's not actually consistent across the codebase. can make it "1" everywhere else as well.

```suggestion
tokio = { version = "1", features = ["sync"] }
```